.github/benchmarks/netbench.json

```diff
@@ -0,0 +1,108 @@
+[
+    {
+        "name": "Netbench TCP BW - Client",
+        "path": "target/x86_64-unknown-hermit/release/tcp-client-bw",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench TCP BW - Server",
+        "path": "target/x86_64-unknown-hermit/release/tcp-server-bw",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench UDP BW - Client",
+        "path": "target/x86_64-unknown-hermit/release/udp-client-bw",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench UDP BW - Server",
+        "path": "target/x86_64-unknown-hermit/release/udp-server-bw",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench TCP Latency - Client",
+        "path": "target/x86_64-unknown-hermit/release/tcp-client-latency",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench TCP Latency - Server",
+        "path": "target/x86_64-unknown-hermit/release/tcp-server-latency",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench UDP Latency - Client",
+        "path": "target/x86_64-unknown-hermit/release/udp-client-latency",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench UDP Latency - Server",
+        "path": "target/x86_64-unknown-hermit/release/udp-server-latency",
+        "group": "File Size",
+        "plot_group": "Netbench Benchmark File Size"
+    },
+    {
+        "name": "Netbench TCP BW - Client",
+        "command": "parallel ::: 'cargo run --manifest-path ./Cargo.toml --bin tcp-server-bw --release --target x86_64-unknown-linux-gnu -- --address 10.0.5.3 --bytes 1048576 --rounds 1000' 'sleep 10 && sudo qemu-system-x86_64 -display none -serial stdio -kernel hermit-loader-x86_64 -cpu qemu64,apic,fsgsbase,rdtscp,xsave,xsaveopt,fxsr,rdrand -enable-kvm -initrd target/x86_64-unknown-hermit/release/tcp-client-bw -smp 2 -m 1024M -netdev user,id=u1,hostfwd=tcp::9975-:9975,hostfwd=udp::9975-:9975,net=192.168.76.0/24,dhcpstart=192.168.76.9 -device virtio-net-pci,netdev=u1,disable-legacy=on,packed=on,mq=on -append \"-- --nonblocking --address 127.0.0.1 --bytes 1048576 --rounds 1000\"'",
```

Comment:
Is there a reason for using user networking instead of taps? I get more than 15% more bandwidth on my machine.

Reply:
It should work as described in the [loader's README](https://github.com/hermit-os/loader?tab=readme-ov-file#network-support). :)